models/post.coffee

```diff
@@ -0,0 +1,14 @@
+mongoose = require('mongoose')
```

Comment:
I'm sorry, that wasn't sarcastic at all. I was referring to the fact that this is a separate file from `server.coffee`, which I keep meaning to break down into smaller components. All good and positive :).